javascript/just.config.cjs

```diff
@@ -98,6 +99,31 @@ function recursiveReplace(obj, pattern, replacement) {
   }
 }
 
+function transformLoadTask() {
+  return async () => {
+    const js = {
+      content:
+        'export * from "./dist/src/load.js";\n//# sourceMappingURL=load.js.map\n',
+    };
+    await writeFile('load.js', js.content);
+
+    const sourceMap = {
```

Comment:
Having this step hardcode the entrypoint, and a sourcemap for it, doesn't seem very maintainable or resilient to future changes (especially since all the tests use the esm version).

I think we would want to include this new file in existing transformation and declaration generation instead.